### PR TITLE
New effects

### PR DESCRIPTION
**Pull request by VALERA771**

### CODE REVIEW

**Comment by louis1706** [comment]
i don't thinks it's needed there no difference between FlashLight and Lantern

File: Exiled.API/Enums/EffectType.cs [hunk 1]
```diff
@@ -199,5 +199,25 @@ public enum EffectType
         /// Teleports the player to the pocket dimension and drains health until the player escapes or is killed.
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Makes walking sound volume equal 0.1.
```
Comment (louis1706) [inline Exiled.API/Enums/EffectType.cs:204]: reduce by 10% the sound volume of walking (this effect is multiply by the intensity)

File: Exiled.API/Extensions/EffectTypeExtension.cs [hunk 2]
```diff
@@ -5,6 +5,8 @@
 // </copyright>
 // -----------------------------------------------------------------------
 
+using InventorySystem.Items.MarshmallowMan;
```
Comment (louis1706) [inline Exiled.API/Extensions/EffectTypeExtension.cs:8]: ????

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Extensions/EffectTypeExtension.cs (hunk 2 above)
Comment (VALERA771) [reply Exiled.API/Extensions/EffectTypeExtension.cs:8]: marshmallow effect there. idk why

Comment (louis1706) [reply Exiled.API/Extensions/EffectTypeExtension.cs:8]: i mean it's not in the right place of using lol

Comment (VALERA771) [reply Exiled.API/Extensions/EffectTypeExtension.cs:8]: yeah. I'll edit it

File: Exiled.API/Enums/EffectType.cs [hunk 3]
```diff
@@ -199,5 +199,25 @@ public enum EffectType
...
+
+        /// <summary>
+        /// The effect that receive player when he getting attacked by SCP-3114 ability.
+        /// </summary>
+        Strangled,
+
+        /// <summary>
+        /// Allows you to bypass doors.
+        /// </summary>
+        Ghoslty,
```
Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:221]:
```suggestion
        Ghostly,
```
typo

File: Exiled.API/Extensions/EffectTypeExtension.cs [hunk 4]
```diff
@@ -66,6 +64,10 @@ public static class EffectTypeExtension
             { EffectType.Traumatized, typeof(Traumatized) },
             { EffectType.AntiScp207, typeof(AntiScp207) },
             { EffectType.Scanned, typeof(Scanned) },
+            { EffectType.SilentWalk, typeof(SilentWalk) },
+            { EffectType.Marshmallow, typeof(MarshmallowEffect) },
+            { EffectType.Strangled, typeof(Strangled) },
+            { EffectType.Ghoslty, typeof(Ghostly) },
```
Comment (Thundermaker300) [inline Exiled.API/Extensions/EffectTypeExtension.cs:70]:
```suggestion
            { EffectType.Ghostly, typeof(Ghostly) },
```
typo

**Review by Thundermaker300 — CHANGES_REQUESTED** [review]

File: Exiled.API/Enums/EffectType.cs [hunk 5]
```diff
@@ -199,5 +199,25 @@ public enum EffectType
...
+        /// </summary>
+        Marshmallow,
+
+        /// <summary>
+        /// The effect that receive player when he getting attacked by SCP-3114 ability.
+        /// </summary>
+        Strangled,
+
+        /// <summary>
+        /// Allows you to bypass doors.
```
Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:219]:
```suggestion
        /// Makes the player nearly invisible, and allows them to pass through doors.
```
Documentation nitpick

File: Exiled.API/Enums/EffectType.cs [hunk 6]
```diff
@@ -199,5 +199,25 @@ public enum EffectType
...
         /// </summary>
         PocketCorroding,
+
+        /// <summary>
+        /// Reduces walking sound by 10%.
+        /// </summary>
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
```
Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:209]: im good with this because it's just funny

File: Exiled.API/Enums/EffectType.cs [hunk 7]
```diff
@@ -199,5 +199,25 @@ public enum EffectType
...
+        /// </summary>
+        Marshmallow,
```
Comment (louis1706) [inline Exiled.API/Enums/EffectType.cs:211]: this need to be puted as Obsolete[] (and make sure to is still use in EffectTypeExtension just supress the Error)

it's will actually not working in game

Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:211]:
```suggestion
        [Obsolete("Not functional in-game")]
        Marshmallow,
```
basically what Yamato is saying is

File: Exiled.API/Enums/EffectType.cs [hunk 8]
```diff
@@ -199,5 +199,26 @@ public enum EffectType
...
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
+        /// </summary>
+        [Obsolete("Not functional in-game")]
+        Marshmallow,
+
+        /// <summary>
+        /// The effect that receive player when he getting attacked by SCP-3114 ability.
```
Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:215]:
```suggestion
        /// The effect that is given to the player when he getting attacked by SCP-3114's Strangle ability.
```

File: Exiled.API/Enums/EffectType.cs [hunk 9]
```diff
@@ -199,5 +199,26 @@ public enum EffectType
...
+        SilentWalk,
+
+        /// <summary>
+        /// Makes you a marshmallow guy.
+        /// </summary>
+        [Obsolete("Not functional in-game")]
+        Marshmallow,
+
+        /// <summary>
+        /// The effect that is given to the player when he getting attacked by SCP-3114's Strangle ability.
```
Comment (Thundermaker300) [inline Exiled.API/Enums/EffectType.cs:215]:
```suggestion
        /// The effect that is given to the player when getting attacked by SCP-3114's Strangle ability.
```

**Pull request merged**